### PR TITLE
Add example of configuring memory for Checkstyle workers

### PR DESCRIPTION
**Pull request by big-guy**

### CODE REVIEW

**Comment by big-guy** [comment]
@bot-gradle test this

File: subprojects/docs/src/docs/userguide/core-plugins/checkstyle_plugin.adoc [hunk 1]
```diff
@@ -122,3 +122,16 @@ include::sample[dir="snippets/codeQuality/codeQuality/kotlin",files="build.gradl
 ====
 
 https://github.com/checkstyle/contribution/tree/master/xsl[View a sample Checkstyle stylesheet.]
+
+[[sec:checkstyle_customize_memory]]
+== Changing the amount of memory given to Checkstyle
```
[the remark this answers is not in the document]
Comment (big-guy) [reply subprojects/docs/src/docs/userguide/core-plugins/checkstyle_plugin.adoc:127]: good idea

**Comment by big-guy** [comment]
@bot-gradle test this

**Review by jvandort — APPROVED** [review]

**Comment by big-guy** [comment]
@bot-gradle test and merge

**Pull request merged**